### DISCUSSION TITLE
How to localize app name for each flavor

### DISCUSSION THREAD

**Discussion by bismarabia** [AngeloAvv/flutter_flavorizr#230] [Q&A]
Let's say we have two flavors for my app, apple and avocado
The app also supports two languages, English and French

1. apple flavor
- English: "Apple app"
- French: "Pomme app"

2. avocado flavor
- English: "Avocado app"
- French: "Avocate app"

How do I localize app name in each flavor?

**Comment by AngeloAvv** [comment]
Hey @bismarabia , after you run flutter_flavorizr, you can localise the names of your app following the native guidelines:

https://docs.flutter.dev/ui/accessibility-and-internationalization/internationalization
https://developer.android.com/guide/topics/resources/localization
https://developer.apple.com/documentation/xcode/localization

**Reply by bismarabia** [reply]
@AngeloAvv  I'm still having difficulties in applying those guidelines with different ios schemes and android flavors.

I would be glad if you provide a hello word project with different flavors (schemes) and localized app name for each flavor.
